emcc.py

```diff
@@ -1886,21 +1900,23 @@ def do_minify(): # minifies the code. this is also when we do certain optimizati
            '--mapFileBaseName', target,
            '--offset', str(0)])
 
-    # Move final output to the js target
-    shutil.move(final, js_target)
```

Comment:
this part seems odd. are we relying on a line-ending conversion tool to move files around for us? if so it should be "convert_line_endings_and_or_move_file" ;)

Can that method only convert line endings, and not move files? I guess you're trying to optimize away a copy, but the common case is to not modify the line endings anyhow? Or, we could have the conversion method always return a new file, and we'd do a move outside it as needed, which is just at worst an extra move operation?